TemplateApplication/Onboarding/InterestingModules.swift

```diff
@@ -39,29 +39,25 @@ struct InterestingModules: View {
             actionText: "INTERESTING_MODULES_BUTTON".moduleLocalized,
             action: {
                 #if targetEnvironment(simulator) && (arch(i386) || arch(x86_64))
```

Comment:
As we moved this to the main OnboardingStack View Builder, we could remove this check here, couldn't we?

Reply:
You're completely right, missed that here.